### ISSUE TITLE
Publish snapm on PyPi

### ISSUE THREAD

**Issue by bmr-cymru** [snapshotmanager/snapm#55]
The name `snapm` appears to be available on PyPi: publish snapm there and allow it to be installed via pip.

See also #48, snapshotmanager/boom#45.

**Comment by bmr-cymru** [comment]
Fixed in v0.3.0 (#59 / 9cc6efa).

**Comment by bmr-cymru** [comment]
```
root@localhost:~# pip install snapm
Collecting snapm
  Downloading snapm-0.3.0-py3-none-any.whl.metadata (53 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.2/53.2 kB 1.4 MB/s eta 0:00:00
Collecting boom-boot>=1.6.4 (from snapm)
  Downloading boom_boot-1.6.4-py3-none-any.whl.metadata (46 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.2/46.2 kB 1.4 MB/s eta 0:00:00
Requirement already satisfied: dbus-python>=1.2.16 in /usr/lib64/python3.12/site-packages (from boom-boot>=1.6.4->snapm) (1.3.2)
Downloading snapm-0.3.0-py3-none-any.whl (68 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 68.6/68.6 kB 1.4 MB/s eta 0:00:00
Downloading boom_boot-1.6.4-py3-none-any.whl (126 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 126.4/126.4 kB 3.6 MB/s eta 0:00:00
Installing collected packages: boom-boot, snapm
Successfully installed boom-boot-1.6.4 snapm-0.3.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
root@localhost:~# snapm snapset create -br upgrade / /home /var
SnapsetName:      upgrade
MountPoints:      /, /home, /var
NrSnapshots:      3
Time:             2024-06-20 16:21:17
UUID:             6a7be03b-329d-5973-9202-8ba5f86b6de4
Status:           Active
Autoactivate:     yes
Bootable:         yes
BootEntries:
  SnapshotEntry:  57dd178
  RevertEntry:    31c7865
root@localhost:~# boom list -o+title
BootID  Version                  Name                     RootDevice                                    Title                                                       
6b99618 6.8.5-301.fc40.x86_64    Fedora Linux             /dev/mapper/fedora-root                       Fedora Linux (6.8.5-301.fc40.x86_64) 40 (Server Edition)    
57dd178 6.8.5-301.fc40.x86_64    Fedora Linux             /dev/fedora/root-snapset_upgrade_1718896877_- Snapshot upgrade 2024-06-20 16:21:17 (6.8.5-301.fc40.x86_64)
31c7865 6.8.5-301.fc40.x86_64    Fedora Linux             /dev/fedora/root                              Revert upgrade 2024-06-20 16:21:17 (6.8.5-301.fc40.x86_64)  
```
